colossalai/fx/passes/algorithms/utils.py

```diff
@@ -1,3 +1,10 @@
+import math
+
+
+def _discretize(mem_unit, values):
+    return [math.ceil(value / mem_unit) for value in values]
+
+
 class Chain:
```

Comment:
These classes in the `utils` file are not really `utils`, they are more like to be in  a file like `operation.py` or `action.py`.